libraries/abstractions/common_io/test/test_iot_internal.h

```diff
@@ -33,6 +33,190 @@
 
 #pragma once
 
+
+/*
+ * Test iteration count macros that platforms should set
+ * in their repective "test_iot_config.h" file. If not set
+ * there, the default value of 1 is set here.
+ */
+
+#ifndef IOT_TEST_COMMON_IO_UART_SUPPORTED
+ #define IOT_TEST_COMMON_IO_UART_SUPPORTED  1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_FLASH_SUPPORTED
+ #define IOT_TEST_COMMON_IO_FLASH_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_WATCHDOG_SUPPORTED
+ #define IOT_TEST_COMMON_IO_WATCHDOG_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_GPIO_SUPPORTED
+ #define IOT_TEST_COMMON_IO_GPIO_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_RTC_SUPPORTED
+ #define IOT_TEST_COMMON_IO_RTC_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_TIMER_SUPPORTED
+ #define IOT_TEST_COMMON_IO_TIMER_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_ADC_SUPPORTED
+ #define IOT_TEST_COMMON_IO_ADC_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_RESET_SUPPORTED
+ #define IOT_TEST_COMMON_IO_RESET_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_PERFCOUNTER_SUPPORTED
+ #define IOT_TEST_COMMON_IO_PERFCOUNTER_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_PWM_SUPPORTED
+ #define IOT_TEST_COMMON_IO_PWM_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_I2C_SUPPORTED
+ #define IOT_TEST_COMMON_IO_I2C_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_TEMP_SENSOR_SUPPORTED
+ #define IOT_TEST_COMMON_IO_TEMP_SENSOR_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_POWER_SUPPORTED
+ #define IOT_TEST_COMMON_IO_POWER_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_BATTERY_SUPPORTED
+ #define IOT_TEST_COMMON_IO_BATTERY_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_EFUSE_SUPPORTED
+ #define IOT_TEST_COMMON_IO_EFUSE_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_SPI_SUPPORTED
+ #define IOT_TEST_COMMON_IO_SPI_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_USB_DEVICE_SUPPORTED
+ #define IOT_TEST_COMMON_IO_USB_DEVICE_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_USB_HOST_SUPPORTED
+ #define IOT_TEST_COMMON_IO_USB_HOST_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_SDIO_SUPPORTED
+ #define IOT_TEST_COMMON_IO_SDIO_SUPPORTED 1
+#endif
+
+#ifndef IOT_TEST_COMMON_IO_I2S_SUPPORTED
+ #define IOT_TEST_COMMON_IO_I2S_SUPPORTED 1
+#endif
+
+
+/*
+ * The variables below are used in the tests. Generally. they should be
+ * set in SET_TEST_IOT_RESET_CONFIG() defined in the respective platform
+ * directory. If not set, the default value set at the variable
+ * definition is used.
+ */
+
+/* Uart */
+extern uint8_t ustestIotUartPort;                       /* the index of the UART that will be tested */
```

Comment:
Since this is a `uint8_t`, it should be named `uctestIotUartPort` according to the [FreeRTOS Coding Style Guide.](https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html)

Reply:
OK, will update it.

Reply:
Quick reminder about this

Reply:
Working on it. Thanks for your reminder.

Reply:
Done.